HW03_YiiP.Rmd

```diff
@@ -0,0 +1,99 @@
+---
+title: "HW03"
+author: "Daniel Krochmal"
+output: github_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+```{r load libraries, echo = FALSE}
+library("ggplot2")
+library("magrittr")
+```
+
+## Purification of bacterial zinc transporter YiiP-antibody complex
+
+YiiP is a 70-kDa, dimeric bacterial zinc transporter. In this assay I attempted to purify YiiP in multicomponent complex that consists of:
+
+* YiiP with fusion partner and in a nanodisc
+* Fab1 - an antibody fragment that specifically recognizes YiiP's fusion partner
+* protein G mutant - that recognizes Fab1 with high affinity
+* Fab2 - another antibody fragment that recognizes Fab1-protein G complex, and contains a mutation preventing it from being recognized by another protein G)
+* nanobody - specific to Fab1 and Fab2
+
+This whole complex is intended for cryo-EM analysis. Although the structure of YiiP transporter has already been obtained with X-ray crystallography, it would be interesting to see if the whole Fab/nanobody/protein G part helps in the analysis of data collected with cryo-EM, by helping in correct orienting of the particles by the analysis software.
+
+The purification was successful, but not very efficient - three large peaks on the right correspond to different antibody complexes, but they miss YiiP protein. Much smaller peaks on the left correspond to the same complexes, but with YiiP being present.
+
+
+```{r load dataset and plot, echo = FALSE}
+
+#Load data and clip the dataset to only contain x values from 8 to 25 (x - elution volume and values from 0 to 8 ml correspond to column's void volume)
+yiip.df <- (
+  read.csv("yiip_size-exclusion.csv", header = TRUE, sep = ",") %>%
+    subset(vol > 8 & vol < 25)
```

Comment:
Good use of the pipe and the subset() function! This way, you can format the dataset and set new limits to only include the x-values you're worried about. That's definitely something I'll have to remember.